doc/lua_api.txt

```diff
@@ -5141,7 +5141,25 @@ This is basically a reference to a C++ `ServerActiveObject`
   keys.
     * bit nr/meaning: 0/up, 1/down, 2/left, 3/right, 4/jump, 5/aux1, 6/sneak,
       7/LMB, 8/RMB
-* `set_physics_override(override_table)`
+* `set_physics_modifier(key_string, modifier_table)`: Adds a physics modifier to
+   the player under the given key. Values from all modifiers apply to the player
+   cumulatively.
```

Comment:
`The product of all modifiers apply to the player.` because `cumulatively` is not precise and may even indicate an addition.